client/a8c-for-agencies/sections/referrals/types.ts

```diff
@@ -9,6 +9,13 @@ interface ReferralPurchaseAPIResponse {
 		revoked_at: string | null;
 	};
 	site_assigned: string;
+	subscription?: {
+		product_name: string;
+		purchase_price: string;
```

Comment:
This is actually a number

```suggestion
		purchase_price: number;
```

Reply:
good catch - updated.